packages/account/src/Components/account-limits/__tests__/account-limits-footer.spec.tsx

```diff
@@ -36,7 +36,7 @@ describe('<AccountLimitsFooterPortal/>', () => {
     });
 
     it('should render AppSettings.Footer components correctly', () => {
-        const { container } = render(<AccountLimitsFooterPortalComponent />);
-        expect(container.getElementsByClassName('dc-app-settings__footer').length).toBe(1);
+        render(<AccountLimitsFooterPortalComponent />);
+        expect(screen.queryByTestId('dt_app_settings_footer')).toHaveTextContent('Learn more about account limits');
```

Comment:
do we really need here testid? maybe we can use just getElementByText? and check of its existing in the document

Reply:
but there's a test case above to check for the text